src/App.css

```diff
@@ -0,0 +1,352 @@
+  font-size: 2.2rem;
+  letter-spacing: 0.2rem;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  text-align: center;
+  margin-top: 0;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 0;
+}
+
+.h1b {
+  font-size: 2rem;
+}
+
+h2 {
+  margin-left: 5%;
+  color: #fff;
+}
+
+p {
+  color: #fff;
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.centered {
+  text-align: center;
+  font-size: 1.2rem;
+}
+
+.under {
+  text-decoration: underline;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+  width: 50px;
+  font-size: 1rem;
+  font-weight: normal;
+  padding-top: 8px;
+  margin-left: 0;
+  padding-left: 0;
+  text-shadow: 1px 1px rgb(158, 158, 158);
+}
+
+a:hover,
+a:active {
+  color: #ffb8e6;
+  font-weight: 600;
+}
+
+.navlink {
+  width: 15%;
+  text-align: center;
+}
+
+.divider {
+  width: 15%;
+}
+
+ul {
+  width: 90%;
+  list-style: none;
+  margin: 10px auto 0 auto;
+  padding: 0;
+  height: 300px;
+}
+
+li {
+  margin: 0 5px 2px 5px;
+  padding-left: 3%;
+  padding-top: 1%;
+  padding-bottom: 1%;
+  border: 1px solid #474747;
+  border-radius: 4px;
+}
+
+li:hover {
+  background-color: rgb(172, 61, 98);
+}
+
+li:nth-child(odd) {
+  background-color: rgb(209, 81, 124);
+}
+
+li:nth-child(odd):hover {
+  background-color: rgb(172, 61, 98);
+}
+
+.white {
+  color: #fff;
+}
+
+.divschoollist {
+  background-color: rgb(252, 89, 143);
+  height: 100vh;
+  overflow-y: auto;
+}
+
+.container {
+  height: 50%;
+  overflow-y: auto;
+}
+
+.divhome {
+  background-color: rgb(252, 89, 143);
+  min-height: 100vh;
+}
+
+.sitename {
+  color: #fff;
+  font-weight: 700;
+  font-size: 2rem;
+  letter-spacing: 0.1rem;
+  text-shadow: 1px 1px rgb(158, 158, 158);
+}
+
+.lato {
+  font-family: 'Lato', sans-serif;
+}
+
+.gillsans {
+  font-family: 'Gill Sans', sans-serif;
+}
+
+.divsearch {
+  display: flex;
+  justify-content: space-evenly;
+  width: 80%;
+  margin: 8px auto 15px auto;
+}
+
+input {
+  font-size: 1.2rem;
+  border-radius: 4px;
+  border: 1px solid #646464;
+  width: 20rem;
+}
+
+button {
+  background-color: #fff;
+  color: #646464;
+  font-size: 1rem;
+  padding: 1px 12px;
+  border-radius: 4px;
+  border: 1px solid #646464;
+}
+
+button:hover {
+  background-color: #d1d1d1;
+  color: #646464;
+  font-weight: bold;
+}
+
+.divdetails {
+  height: 100%;
+}
+
+.subtitles {
+  color: #ffb8e6;
+}
+
+.biggertext {
+  font-size: 2rem;
+}
+
+.grid {
+  width: 90%;
+  margin-right: auto;
+  margin-left: auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.schoolname {
+  font-size: 1.5rem;
+}
+
+.boroheading {
+  font-size: 2rem;
+  letter-spacing: 0.2rem;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 8px;
+  text-shadow: 1px 1px rgb(158, 158, 158);
+}
+
+.imgcard {
+  display: flex;
+  flex-direction: column;
+}
+
+.imagek:hover,
+.imagem:hover,
+.imagex:hover,
+.imageq:hover,
+.imager:hover {
+  background-color: #ffb8e6;
+  border-radius: 8px;
+  cursor: pointer;
+}
```

Comment:
- [ ] You did a great work however it is required to follow the given design. Kindly use two different shades of the same colors and alternate them for your elements as in the design:


<img width="433" alt="Screenshot 2023-10-20 at 9 29 05 PM" src="https://github.com/peter-mwovi/metrics_webapp/assets/53347966/40caa0fd-f9a0-429a-83f0-14aa3a35a4d5">